swift_scality_backend/server.py

```diff
@@ -34,6 +35,19 @@
 FRAG_PREFS_STUB = object()
 
 
+def log_to_file(conf, name, log_to_console, log_route, fmt, logger, adapted_logger):
```

Comment:
Would it make sense to have this in the `utils` module rather than `server`?

Reply:
Right, done!